### PR TITLE
IBP-5311 IBP-5379 Remove authToken usages and references

### PR DESCRIPTION
**Pull request by abatac**

### CODE REVIEW

File: src/main/java/org/generationcp/breeding/manager/util/Util.java [hunk 1]
```diff
@@ -296,20 +296,4 @@ public static String generateListFolderPathLabel(GermplasmListManager germplasmL
 		 return germplasmListsMap;
 	 }
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/breeding/manager/util/Util.java:298]: Apply code cleanup

File: src/main/java/org/generationcp/ibpworkbench/controller/UserProgramController.java [hunk 2]
```diff
@@ -46,7 +46,7 @@
         final Project project = this.workbenchDataManager.getProjectByUuid(programUUID);
         if (project != null) {
             org.generationcp.commons.util.ContextUtil.setContextInfo(this.request,
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/ibpworkbench/controller/UserProgramController.java:48]: Apply cleanup

File: src/main/java/org/generationcp/ibpworkbench/sample/SampleInfoComponent.java [hunk 3]
```diff
@@ -195,9 +195,8 @@ private static ExternalResource getURLStudy(final Integer studyId, final String
 	}
 
 	private static String getAuthParams(final ContextUtil contextUtil) {
-		final String authToken = contextUtil.getContextInfoFromSession().getAuthToken();
 		return "loggedInUserId=" + contextUtil.getContextInfoFromSession().getLoggedInUserId() + "&selectedProjectId=" + contextUtil
-			.getContextInfoFromSession().getSelectedProjectId() + "&authToken=" + (authToken != null ? authToken : "");
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/ibpworkbench/sample/SampleInfoComponent.java:200]: Apply cleanup

File: src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java [hunk 4]
```diff
@@ -55,8 +55,8 @@ public void onAuthenticationSuccess(final HttpServletRequest request, final Http
 		final boolean shouldShowReleaseNote = this.releaseNoteService.shouldShowReleaseNote(user.getUserid());
 
 		// Initialize the ContextInfo to set the userId of the authenticated user.
-		// The projectId and token will be populated later when a program is opened/loaded.
```
Comment (darla-leafnode) [inline src/main/java/org/generationcp/ibpworkbench/security/WorkbenchAuthenticationSuccessHandler.java:58]: Apply code cleanup (with unused import)

**Review by nahuel-soldevilla — APPROVED** [review]

**Pull request merged**